cacheops/cleaner.py

```diff
@@ -0,0 +1,58 @@
+import logging
```

Comment:
Let's give a tribute to @nicwolff but saving his name `reaper`. Also, I simply like distinctive names)